[2022-02-10 15:45:57,820] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: ex_1_data_ingestion_gcs_dag_v3.download_dataset_task scheduled__2019-10-01T00:00:00+00:00 [queued]>
[2022-02-10 15:45:57,863] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: ex_1_data_ingestion_gcs_dag_v3.download_dataset_task scheduled__2019-10-01T00:00:00+00:00 [queued]>
[2022-02-10 15:45:57,866] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-10 15:45:57,888] {taskinstance.py:1239} INFO - Starting attempt 2 of 2
[2022-02-10 15:45:57,890] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-10 15:45:57,918] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): download_dataset_task> on 2019-10-01 00:00:00+00:00
[2022-02-10 15:45:57,933] {standard_task_runner.py:52} INFO - Started process 1451 to run task
[2022-02-10 15:45:57,947] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'ex_1_data_ingestion_gcs_dag_v3', 'download_dataset_task', 'scheduled__2019-10-01T00:00:00+00:00', '--job-id', '901', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_hw_1_ex.py', '--cfg-path', '/tmp/tmp8p2mzdnw', '--error-file', '/tmp/tmp176hq5b5']
[2022-02-10 15:45:57,950] {standard_task_runner.py:77} INFO - Job 901: Subtask download_dataset_task
[2022-02-10 15:45:58,075] {logging_mixin.py:109} INFO - Running <TaskInstance: ex_1_data_ingestion_gcs_dag_v3.download_dataset_task scheduled__2019-10-01T00:00:00+00:00 [running]> on host 276a0ad90304
[2022-02-10 15:45:58,172] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-10 15:45:58,201] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=ex_1_data_ingestion_gcs_dag_v3
AIRFLOW_CTX_TASK_ID=download_dataset_task
AIRFLOW_CTX_EXECUTION_DATE=2019-10-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-10-01T00:00:00+00:00
[2022-02-10 15:45:58,204] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-02-10 15:45:58,208] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'curl -sS https://s3.amazonaws.com/nyc-tlc/trip+data/yellow_tripdata_2019-10.csv > /opt/***/yellow_tripdata_2019-10.csv']
[2022-02-10 15:45:58,235] {subprocess.py:85} INFO - Output:
[2022-02-10 15:46:27,463] {subprocess.py:93} INFO - Command exited with return code 0
[2022-02-10 15:46:27,574] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=ex_1_data_ingestion_gcs_dag_v3, task_id=download_dataset_task, execution_date=20191001T000000, start_date=20220210T154557, end_date=20220210T154627
[2022-02-10 15:46:27,639] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-10 15:46:27,724] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-02-10 16:52:45,668] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: ex_1_data_ingestion_gcs_dag_v3.download_dataset_task scheduled__2019-10-01T00:00:00+00:00 [queued]>
[2022-02-10 16:52:45,692] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: ex_1_data_ingestion_gcs_dag_v3.download_dataset_task scheduled__2019-10-01T00:00:00+00:00 [queued]>
[2022-02-10 16:52:45,694] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-10 16:52:45,696] {taskinstance.py:1239} INFO - Starting attempt 2 of 2
[2022-02-10 16:52:45,698] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-10 16:52:45,728] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): download_dataset_task> on 2019-10-01 00:00:00+00:00
[2022-02-10 16:52:45,742] {standard_task_runner.py:52} INFO - Started process 1470 to run task
[2022-02-10 16:52:45,750] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'ex_1_data_ingestion_gcs_dag_v3', 'download_dataset_task', 'scheduled__2019-10-01T00:00:00+00:00', '--job-id', '1011', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_hw_1_ex.py', '--cfg-path', '/tmp/tmpy66jrnvg', '--error-file', '/tmp/tmp_2jvko7e']
[2022-02-10 16:52:45,753] {standard_task_runner.py:77} INFO - Job 1011: Subtask download_dataset_task
[2022-02-10 16:52:46,019] {logging_mixin.py:109} INFO - Running <TaskInstance: ex_1_data_ingestion_gcs_dag_v3.download_dataset_task scheduled__2019-10-01T00:00:00+00:00 [running]> on host 106a34111faf
[2022-02-10 16:52:46,177] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-10 16:52:46,256] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=ex_1_data_ingestion_gcs_dag_v3
AIRFLOW_CTX_TASK_ID=download_dataset_task
AIRFLOW_CTX_EXECUTION_DATE=2019-10-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-10-01T00:00:00+00:00
[2022-02-10 16:52:46,258] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-02-10 16:52:46,261] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'curl -sS https://s3.amazonaws.com/nyc-tlc/trip+data/yellow_tripdata_2019-10.csv > /opt/***/yellow_tripdata_2019-10.csv']
[2022-02-10 16:52:46,291] {subprocess.py:85} INFO - Output:
[2022-02-10 16:53:17,963] {subprocess.py:93} INFO - Command exited with return code 0
[2022-02-10 16:53:18,524] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=ex_1_data_ingestion_gcs_dag_v3, task_id=download_dataset_task, execution_date=20191001T000000, start_date=20220210T165245, end_date=20220210T165318
[2022-02-10 16:53:18,842] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-10 16:53:19,038] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
